workers/unity/Packages/com.improbable.gdk.tools/.CodeGenerator/GdkCodeGenerator/Templates/UnityComponentDataGenerator.tt

```diff
@@ -173,6 +184,16 @@ var fieldDetails = fieldDetailsList[i]; #>
 
         public static class Serialization
         {
+            public static void SerializeData(<#= componentNamespace #>.Component component, global::Improbable.Worker.Core.SchemaObject obj, global::Unity.Entities.World world)
```

Comment:
Perhaps `SerializeSnapshot` to be consistent with `DeserializeSnapshot`?

Reply:
This doesn't serialize the snapshot 😛

Reply:
I would call this SerializeComponent to be consistent.